support react 0.14

0x80 opened the pull request.

contra commented:
this is excluding 0.12

0x80 commented:
Didn't think about it. Are people still using that? :) Anyway I've added it.

contra commented:
@0x80 Are you 100% sure that 0.14 doesn't fall within *? You may have been on a pre release

0x80 commented:
No not 100% sure. When I was trying to fix the package.json npm kept giving me errors until I removed my node_modules and installed again. I'm not sure what got messed up there. But I've never seen a package depend on *. 

At first I had added react 0.14 to my package.json and tried a clean install with your 0.0.8. Your package failed as the only one in my dependencies, so I'm pretty sure it wasn't compatible.
